lib/CohereEmbedV3ServicesStackProps.ts

```diff
@@ -52,6 +52,10 @@ export interface CohereEmbedV3Services {
      * The memory type for the deployment.
      */
     readonly cdkDeployMemoryType: string;
+    /**
+     * The vpcId for the deployment.
+     */
+    readonly vpcId: string;
```

Comment:
**Suggestion:** Add a description to the `vpcId` property that explains its purpose and format. This will improve code documentation and make it easier for other developers to understand the expected input.
> 🌎  **Medium Priority** 
 > [category: **`documentation`**, importance: **`6`**]
> **Reason:** This suggestion improves code documentation, making it easier for developers to understand and use the vpcId property correctly.
 

```suggestion
    /**
     * The ID of the VPC to deploy the services into.
     * This should be in the format 'vpc-xxxxxxxx' where x is a hexadecimal digit.
     * The VPC must already exist in the AWS account and region where the stack is being deployed.
     */
    readonly vpcId: string;
```